controllers/tasks.js

```diff
@@ -401,6 +401,27 @@ const assignTask = async (req, res) => {
   }
 };
 
+const updateOldTaskStatus = async (req, res) => {
+  try {
+    const allTasks = await tasks.fetchTasks();
+    const allOldTasks = allTasks.filter(
+      (task) => task.status.toUpperCase() === "COMPLETED" || task.status === "unassigned"
+    );
+    const updatedTasks = [];
+    for (const task of allOldTasks) {
+      if (task.status.toUpperCase() === "COMPLETED") {
+        updatedTasks.push(await tasks.updateTask({ status: "DONE" }, task.id));
+      } else if (task.status === "unassigned") {
+        updatedTasks.push(await tasks.updateTask({ status: "UNASSIGNED" }, task.id));
+      }
+    }
+    return res.json({ message: `Updated Old tasks`, tasks: updatedTasks });
```

Comment:
can we provide a more meaningful response message please

Reply:
<img width="500" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/77037622/6da5ea2d-3a68-42ac-951d-a029ce41d125">

Changed it.